neuro-sdk/src/neuro_sdk/_clusters.py

```diff
@@ -0,0 +1,117 @@
+from neuro_config_client import AzureReplicationType as _AzureReplicationType
+from neuro_config_client import AzureStorage as _AzureStorage
+from neuro_config_client import AzureStorageOptions as _AzureStorageOptions
+from neuro_config_client import AzureStorageTier as _AzureStorageTier
+from neuro_config_client import CloudProvider as _CloudProvider
+from neuro_config_client import CloudProviderOptions as _CloudProviderOptions
+from neuro_config_client import CloudProviderType as _CloudProviderType
+from neuro_config_client import Cluster as _ConfigCluster
+from neuro_config_client import ClusterStatus as _ClusterStatus
+from neuro_config_client import ConfigClientBase
+from neuro_config_client import EFSPerformanceMode as _EFSPerformanceMode
+from neuro_config_client import EFSThroughputMode as _EFSThroughputMode
+from neuro_config_client import GoogleCloudProvider as _GoogleCloudProvider
+from neuro_config_client import GoogleFilestoreTier as _GoogleFilestoreTier
+from neuro_config_client import GoogleStorage as _GoogleStorage
+from neuro_config_client import GoogleStorageOptions as _GoogleStorageOptions
+from neuro_config_client import NodePool as _NodePool
+from neuro_config_client import NodePoolOptions as _NodePoolOptions
+from neuro_config_client import OnPremCloudProvider as _OnPremCloudProvider
+from neuro_config_client import ResourcePreset as _ResourcePreset
+from neuro_config_client import Storage as _Storage
+from neuro_config_client import StorageInstance as _StorageInstance
+from neuro_config_client import StorageOptions as _StorageOptions
+from neuro_config_client import TPUPreset as _TPUPreset
+from neuro_config_client import VCDCloudProvider as _VCDCloudProvider
+from neuro_config_client import VCDCloudProviderOptions as _VCDCloudProviderOptions
+from neuro_config_client import VCDStorage as _VCDStorage
+
+from ._config import Config
+from ._core import _Core
+from ._rewrite import rewrite_module
+from ._utils import NoPublicConstructor
+
+# Explicit __all__ to re-export neuro_config_client entities
+
+__all__ = [
+    "_AWSCloudProvider",
+    "_AWSStorage",
+    "_AWSStorageOptions",
+    "_AzureCloudProvider",
+    "_AzureReplicationType",
+    "_AzureStorage",
+    "_AzureStorageOptions",
+    "_AzureStorageTier",
+    "_CloudProvider",
+    "_CloudProviderOptions",
+    "_CloudProviderType",
+    "_ClusterStatus",
+    "_Clusters",
+    "_ConfigCluster",
+    "_EFSPerformanceMode",
+    "_EFSThroughputMode",
+    "_GoogleCloudProvider",
+    "_GoogleFilestoreTier",
+    "_GoogleStorage",
+    "_GoogleStorageOptions",
+    "_NodePool",
+    "_NodePoolOptions",
+    "_OnPremCloudProvider",
+    "_ResourcePreset",
+    "_Storage",
+    "_StorageInstance",
+    "_StorageOptions",
+    "_TPUPreset",
+    "_VCDCloudProvider",
+    "_VCDCloudProviderOptions",
+    "_VCDStorage",
+]
+
+
+@rewrite_module
+class _Clusters(ConfigClientBase, metaclass=NoPublicConstructor):
```

Comment:
Are you sure that the inheritance from `ConfigClientBase` is the best solution here?
Do we need to expose all `ConfigClientBase` methods in SDK?
I would embed it rather than inherit and provide a simple subset of required API methods.

Reply:
I just used the same approach we have in for `_Admin`